privacyidea/static/css/content.css

```diff
@@ -79,6 +79,10 @@ input[type="checkbox"] {
     margin-top: 5px;
 }
 
+.pl-0px {
+    padding-left: 0px;
```

Comment:
Why do we need a new class here for padding this very widget?
Isn't there a generic rule?